src/App.jsx

```diff
@@ -1,31 +1,37 @@
 import './App.css';
```

Comment:
good practice for import order ->

React , npm packages, component, utils, constact, css -> in that order.